app/views/finders/_option_select_facet.html.erb

```diff
@@ -1,8 +1,8 @@
-<% add_app_component_stylesheet("option-select") %>
+<% add_gem_component_stylesheet("option-select") %>
 <% add_gem_component_stylesheet("checkboxes") %>
 
```

Comment:
You should be ok to delete both of these helpers `add_gem_component_stylesheet("option-select")` and `add_gem_component_stylesheet("checkboxes")` because the `option-select` stylesheet is included [here](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_option_select.html.erb#L2-L3) and `checkboxes` stylesheet [here](https://github.com/alphagov/govuk_publishing_components/blob/main/app/views/govuk_publishing_components/components/_checkboxes.html.erb#L2).

I've double checked running locally and both stylesheets are included when you remove them from the finder frontend view:

<img width="1435" alt="Screenshot 2023-09-20 at 16 18 48" src="https://github.com/alphagov/finder-frontend/assets/87758239/73920986-afd0-4328-aa4c-b28a01d6f319">

Reply:
Thanks @jon-kirwan, have removed 👍